### PR TITLE
perftest appgw

### PR DESCRIPTION
**Pull request by naikajah**
**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: azure-pipelines.yml [hunk 1]
```diff
@@ -68,6 +68,10 @@ stages:
             parameters:
               environment: 'stg'
               resource_key: 'appgw'
+          - template: devops-templates/terraform-plan-template.yaml
+            parameters:
+              environment: 'perftest'
```
Comment (timja) [inline azure-pipelines.yml]: this needs to be test
```suggestion
              environment: 'test'
```

Comment (timja) [reply azure-pipelines.yml]: this should be renamed as well:
https://github.com/hmcts/cnp-aks-rbac-pipelines/blob/master/terraform/perftest.tfvars
to test.tfvars

Comment (naikajah) [reply azure-pipelines.yml]: done

File: terraform/test.tfvars [hunk 2]
```diff
@@ -1,2 +1,2 @@
-env                       = "perftest"
+env                       = "test"
```
Comment (timja) [inline terraform/test.tfvars]:
```suggestion
env                       = "perftest"
```

**Pull request merged**